looprpc/server.proto

```diff
@@ -74,4 +78,4 @@ message ServerLoopInQuoteResponse {
     uint64 min_swap_amount = 4;
     uint64 max_swap_amount = 5;
     int32 cltv_delta = 6;
-}
\ No newline at end of file
+}
```

Comment:
`max_swap_wait_time_sec` needs to be added to the quote call too, to be able to see the difference in fee